src/api/membersApi.ts

```diff
@@ -0,0 +1,8 @@
+import { MemberType } from '../model/Member';
+import { getSupabaseClient } from './supabase';
+
+export const getMembers = async () => {
+  const { data } = await getSupabaseClient().from('members').select('*');
+
+  return data as MemberType[];
```

Comment:
Be careful what you just did there. You are lying to typescript. The type is different

<img width="236" alt="CleanShot 2024-12-21 at 21 18 08@2x" src="https://github.com/user-attachments/assets/ea0ade69-eee0-4016-9380-8f5cc914041a" />


it can be null. We should handle it via `return data || []`